README.md

```diff
@@ -1,54 +1,316 @@
+instance, if you want to use "japanese" instead of "ja" and "french" instead of
+"fr", then you should configure WOVN.php as below.
+```
+custom_lang_aliases[ja] = japanese
+custom_lang_aliases[fr] = french
+```
+
+**Note for path URL pattern users:**
+You need to update your `.htacces` or Nginx configuration accordingly. For the
+example above, `|ja|` and `|fr|` would become `|japanese|` and `|french|`
+respectively in the expression
+`ar|eu|bn|bg|ca|zh-CHS|zh-CHT|da|nl|en|fi|fr|gl|de|el|he|hu|id|it|ja|ko|lv|ms|my|ne|no|fa|pl|pt|ru|es|sw|sv|tl|th|hi|tr|uk|vi`.
+
+#### `query`
+This parameter tells WOVN.php which query parameters make pages unique. Indeed,
+by default WOVN.io ignores query parameters when searching a translated page.
+If you've created pages on WOVN.io with specific query parameters, you should
+add those query parameters to WOVN.php settings.
+
+For instance, if you have all three pages `https://my-website.com/index.php`,
+`https://my-website.com/index.php?login=1` and
+`https://my-website.com/index.php?forgot_password=1` on WOVN.io, then you should
+configure WOVN.php as below.
+```
+query[] = login
+query[] = forgot_password
+```
+
+#### `ignore_paths`
+This parameter tells WOVN.php to not localize content withing given directories.
+
+For instance, if you want to not localize the admin panels of your website, you
+should configure WOVN.php as below. WOVN.php will localize
+`https://my-website.com/index.html` but not `https://my-wesite.com/admin/` nor
+`https://my-website.com/admin/plugin.html`.
+```
+ignore_paths[] = /admin
+```
+
+#### `ignore_regex`
+This parameter is similar to `ignore_paths` (see [above](#ignore_paths)) except
+that you can use regular expressions instead.
+
+For instance, if you want to not localize the search pages, you should configure
+WOVN.php as below. WOVN.php will localize
+`https://my-website.com/search/index.php` but not
+`https://my-website.com/search/01/` nor `https://my-website.com/search/02/`.
 ```
-require: {
-    "WOVNio/WOVN.php": "*"
-}
+ignore_paths[] = /\/search\/\d\d\//
 ```
-### 2. Edit the configuration file
-These are the two main settings for your project
- - project_token: your WOVNio's project token
- - url_pattern: query, path or subdomain (default is query)
- - default_lang: the code of the default language of your website, ex: ja (for Japanese)
- - encoding: encoding of html. If not specified, detect automatically.
-     Supported encoding: 'UTF-8', 'EUC-JP', 'SJIS', 'eucJP-win', 'SJIS-win', 'JIS', 'ISO-2022-JP', 'ASCII'
 
-Please copy the sample file from the library `wovn.ini.sample`, rename it as `wovn.ini` and paste it on your framework root directory (same level as WOVN.php directory)
--->
+#### `ignore_class`
+This parameter tells WOVN.php which HTML fragments it should ignore when
+localizing. The classes given by `ignore_class` are HTML element classes. Every
+HTML element of the given classes won't be translated by WOVN.php.
```

Comment:
> ~Every HTML element of the given classes won't be translated by WOVN.php.~
> All HTML elements with one or more ignored class will be ignored by WOVN.php.

Reply:
Partially done. I kept the end of the sentence as is because I want it to be clear than ignoring is not translating.